data/nlu/responses.md

```diff
@@ -72,8 +72,8 @@
 
 ## chitchat
 * chitchat/ask_builder 
-  -  The engineers at Rasa 🛠
   -  One of the smart engineers at Rasa! 🧠
+  -  The engineers at Rasa 🛠
```

Comment:
Since people often ask about mother and father, figured the singular one made more sense. Also i miss our varied clever responses 😭

Reply:
i am confused, how does having two responses here even work? Or are they just ignored?

Reply:
All but the first are ignored

Reply:
It gets quite awkward when users ask for multiple jokes 🙈

Reply:
ugh